### DISCUSSION TITLE
How to hide BETA software banner?

### DISCUSSION THREAD

**Discussion by emorgoch** [sct/overseerr#1900] [Q&A] [answered]
I'm currently running v1.25.0, and it's working great for me. However, one thing that annoys me is the full width banner at the top of the page saying that the software is BETA. Is there any way to hide the BETA Software banner, short of using some CSS add-ins to modify and hide it?

**Comment by TheCatLady** [comment, marked as answer]
No, there is no way to hide the banner aside from using external tools like ad blockers.

However, note that the banner is only displayed for admin users.
